src/patterns/molecules/notification-bar/notification-bar.hbs

```diff
@@ -0,0 +1,90 @@
+---
+name: Notification bar
+description: A notification bar provides contextual feedback messages
+notes: |
+    - Notification bars have a range of different colors
+    - `mzp-t-success`
+    - `mzp-t-warning`
+    - `mzp-t-error`
+    - `mzp-t-click`
+    - An optional dismiss button
+    - `mzp-has-dismiss`
+    - And optional stickiness
+    - `mzp-is-sticky`
+order: 5
+---
+
+<button class="mzp-c-button">Open Notification</button>
+
+<ul class="notification-demo-list">
+  <li>
+    <aside class="mzp-c-notification-bar mzp-t-success">
+      <p class="mzp-c-notification-bar-content">Great! You clicked the thing!</p>
+      <button class="mzp-c-notification-bar-button" type="button"></button>
+    </aside>
+  </li>
+
+  <li>
+    <aside class="mzp-c-notification-bar mzp-t-warning">
+      <p class="mzp-c-notification-bar-content">Warning, you're about to do something bad!</p>
+      <button class="mzp-c-notification-bar-button" type="button"></button>
+    </aside>
+  </li>
+
+  <li>
+    <aside class="mzp-c-notification-bar mzp-t-error">
+      <p class="mzp-c-notification-bar-content">Whoops. You deleted something.</p>
+      <button class="mzp-c-notification-bar-button" type="button"></button>
+    </aside>
+  </li>
+
+  <li>
+    <aside class="mzp-c-notification-bar mzp-t-click">
+      <p class="mzp-c-notification-bar-content">Click this thing.</p>
+      <button class="mzp-c-notification-bar-button" type="button"></button>
+    </aside>
+  </li>
+
+  <li>
+    <aside class="mzp-c-notification-bar">
+      <p class="mzp-c-notification-bar-content">Default.</p>
+      <button class="mzp-c-notification-bar-button" type="button"></button>
+    </aside>
+  </li>
+</ul>
+
+<script src="{{@root.baseurl}}/assets/protocol/protocol/js/protocol-notification-bar.js"></script>
+
+<script>
+  (function () {
+    'use strict';
+
+    var dissmissButtons = document.querySelectorAll('.mzp-c-notification-bar-button');
+    for (var i = 0; i < dissmissButtons.length; i++) {
+      dissmissButtons[i].addEventListener('click', function(e) {
+        e.currentTarget.parentNode.remove();
+      });
+    }
+
+    var notificationButton = document.querySelector('.mzp-c-button');
+
+    notificationButton.addEventListener('click', function (e) {
+      e.preventDefault();
+
+      Mzp.Notification.init(e.target, content, {
```

Comment:
The demo is throwing an error when I click the button saying `ReferenceError: content is not defined`.

Looking at the code, `content` does seem to be undefined here, but also the JS does nothing with this value, so is it needed? If I remove this it seems to work ok.

I also notice you're passing `e.target` as the origin here, but you're not using that in the code either. It might be useful to return focus to the origin element when the notification is closed though? :)